configs/detection/detr/detr.py

```diff
@@ -1,8 +1,7 @@
 # model settings
 model = dict(
     type='Detection',
-    pretrained=
-    'http://pai-vision-data-hz.oss-cn-zhangjiakou.aliyuncs.com/pretrained_models/easycv/resnet/torchvision/resnet50.pth',
+    pretrained=True,
```

Comment:
if user set pretrained=True, and the model does not have a default url, an error should be raised directly

Reply:
done